MacDown/Code/Document/MPDocument.m

```diff
@@ -536,6 +536,8 @@ - (BOOL)readFromData:(NSData *)data ofType:(NSString *)typeName
 - (BOOL)prepareSavePanel:(NSSavePanel *)savePanel
 {
     savePanel.extensionHidden = NO;
+    [savePanel setCanSelectHiddenExtension:NO];
```

Comment:
@uranusjr totally. I first set with the dot property syntax but it was read only. Of course I must've by mistake been trying to set another property, don't know which one. Because what you suggested works and the property isn't read only 👍  Pushed a commit with this change.